docs/ru/assemble_4.md

```diff
@@ -0,0 +1,336 @@
+
+    <center>
+        <img src="../assets/4/07.png" width=300 class="zoom border">
+        <img src="../assets/4/07_1.png" width=300 class="zoom border">
+    </center>
+
+## Монтаж платы распределения питания
+
+1. [Залудите](tinning.md) контактные площадки платы питания.
+2. С помощью мультиметра проверьте отсутствие контактного замыкания (прозвонить):
+    * Выставите мультиметр в режим прозвона.
+    * Проверьте работу мультиметра путем замыкания щупов между собой. При корректной работе прибор издаст характерный звук.
+    * Попарно один щуп прикладывается к контакту **«+»**, а второй к **«-»/GND**. Если в цепи есть короткое замыкание, издается звук.
+
+## **Монтаж PDB**
+
+1. Установите 4 стойки "папа-мама" 6 мм на центральную деку винтами М3х6.
+
+    <center>
+        <img src="../assets/4/08.png" width=300 class="zoom border">
+        <img src="../assets/4/08_1.png" width=300 class="zoom border">
+    </center>
+
+2. Установите PDB на стойки.
+3. Стрелки на PDB и центральной деке должны быть направлены в одну сторону.
+
+    <img src="../assets/4/09.png" width=300 class="zoom center border">
+
+## Пайка регуляторов и ВЕС
+
+1. Припаяйте фазные провода моторов к регуляторам.
+2. Припаяйте силовые провода регуляторов к контактным площадкам платы (**<font color=red>красный</font>** к **«+»**, **черный** к **«-»**).
+
+    <center>
+        <img src="../assets/4/09.png" width=300 class="zoom border">
+        <img src="../assets/4/10_1.png" width=300 class="zoom border">
+    </center>
+
+3. Припаяйте силовые провода каждого BEC к контактным площадкам одного из регуляторов (**<font color=red>красный</font>** к **«+»**, **черный** к **«-»**).
+
+    <center>
+        <img src="../assets/4/10.png" width=300 class="zoom border">
+        <img src="../assets/4/12_1.png" width=300 class="zoom border">
+    </center>
+
+## Сопряжение приёмника и пульта
+
+1. Подключите радиоприемник к разъему от BEC 5В.
+2. Установите джампер на вход (B/VCC).
+3. Подключите АКБ.
+4. Светодиод на радиоприемнике должен мигать.
+
+    <center>
+        <img src="../assets/4/13.png" width=300 class="zoom border">
+        <img src="../assets/4/13_1.png" width=300 class="zoom border">
+    </center>
+
+5. Зажмите кнопку На пульте зажать кнопку *BIND KEY*.
+6. Включите пульт (перещелкните *POWER*, не отпуская *BIND KEY*).
+
+    <img src="../assets/4/radio/binding.png" class="zoom center">
+
+7. Ждите синхронизации.
+8. Отсоедините джампер.
+9. Светодиод на приемнике должен гореть непрерывно.
+
+Если пульт заблокирован, необходимо перевести все стики в начальное положение:
+
+1. Левый стик в **центральной нижней позиции**.
+2. Правый стик в **центре**.
+3. Переключатели A, B, C, D в положение **"от себя"**.
+
+    <img src="../assets/4/radio/base.png" alt="Нейтральное положение стиков пульта" class="zoom center">
+
+Убедитесь, что PPM в меню RX Setup отключен:
```

Comment:
Why do we do that? I guess there should be something about checking the motor rotation direction, but there's nothing about that below; besides, we still need to use PPM with the flight controller.